server/channels/auth.js

```diff
@@ -48,13 +62,48 @@ const register = async (socket, data) => {
   emit(socket, AUTH_REGISTERED, { registered: true });
 };
 
+const createGenfors = async (socket, data) => {
+  const { password, title, date } = data;
+  if (verifyAdminPassword(password)) {
+    await addGenfors(title, date);
+    logger.info('Created genfors by administrative request.', { title });
+  } else {
+    logger.warn('Someone tried to authenticate as administrator.', { title });
+    emit(socket, 'AUTH_ERROR', { error: 'Ugyldig administratorpassord.' });
+  }
+};
+
 const listener = (socket) => {
   async function action(data) {
     switch (data.type) {
       case AUTH_REGISTER: {
         register(socket, data);
         break;
       }
+      case ADMIN_CREATE_GENFORS: {
```

Comment:
Isn't it a bit odd to have ADMIN_CREATE_GENFORS in the auth channel?

Reply:
This is basically an extension of ADMIN_LOGIN which in addition takes the title and date arguments. I think it fits better here than to pollute the meeting channel with password and admin authentication logic.

Reply:
That makes sense.

Reply:
I'll look into it though.